standard/documentation-comments.md

```diff
@@ -849,130 +893,163 @@ The following example shows the source code of a Point class:
+        /// the given x- and y-offsets.
+        /// <example>For example:
+        /// <code>
+        /// Point p = new Point(3,5);
+        /// p.Translate(-1,3);
+        /// </code>
+        /// results in <c>p</c>'s having the value (2,8).
+        /// <see cref="Move"/>
+        /// </example>
+        /// </summary>
+        /// <param><c>xor</c> is the relative x-offset.</param>
+        /// <param><c>yor</c> is the relative y-offset.</param>
+        public void Translate(int xor, int yor)
+        {
+            X += xor;
+            Y += yor;
+        }
+        
+        /// <summary>
+        /// This method determines whether two Points have the same location.
+        /// </summary>
+        /// <param>
+        /// <c>o</c> is the object to be compared to the current object.
+        /// </param>
+        /// <returns>
+        /// True if the Points have the same location and they have
+        /// the exact same type; otherwise, false.
+        /// </returns>
+        /// <seealso cref="operator=="/>
+        /// <seealso cref="operator!="/>
+        public override bool Equals(object o)
+        {
+            if (o == null)
+            {
+                return false;
+            }
+            if (this == o) 
+            {
+                return true;
+            }
+            if (GetType() == o.GetType()) 
+            {
+                Point p = (Point)o;
+                return (X == p.X) && (Y == p.Y);
+            }
             return false;
-         }
-         if (this == o) {
-            return true;
-         }
-         if (GetType() == o.GetType()) {
-            Point p = (Point)o;
-            return (X == p.X) && (Y == p.Y);
-         }
-         return false;
-      }
-
-      /// <summary>Report a point's location as a string.</summary>
-      /// <returns>A string representing a point's location, in the form (x,y),
-      /// without any leading, training, or embedded whitespace.</returns>
-      public override string ToString() {
-         return "(" + X + "," + Y + ")";
-      }
-
-      /// <summary>This operator determines whether two Points have the same
-      /// location.</summary>
-      /// <param><c>p1</c> is the first Point to be compared.</param>
-      /// <param><c>p2</c> is the second Point to be compared.</param>
-      /// <returns>True if the Points have the same location and they have
-      /// the exact same type; otherwise, false.</returns>
-      /// <seealso cref="Equals"/>
-      /// <seealso cref="operator!="/>
-      public static bool operator==(Point p1, Point p2) {
-         if ((object)p1 == null || (object)p2 == null) {
+        }
+        
+        /// <summary>Report a point's location as a string.</summary>
+        /// <returns>
+        /// A string representing a point's location, in the form (x,y),
+        /// without any leading, training, or embedded whitespace.
+        /// </returns>
+        public override string ToString() => 
```

Comment:
Keep the implementation on the same line as the declaration, as it'll still be quite short?
(I agree with the formatting for expression-bodied members that need the body on a new line, btw.)

Of course, the implementation would be even shorter as `$"({X}, {Y})"` :)